frontendservice/public/index.html

```diff
@@ -11,24 +11,36 @@
 
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="utf-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <link rel="icon" href="<%= BASE_URL %>favicon.ico">
+  <head>
+    <style>
+      body {
+        background-image: url("https://img.freepik.com/free-photo/black-friday-composition-with-four-bags-cart_23-2147709334.jpg?size=626&ext=jpg");
+        background-repeat: no-repeat;
+        background-size: cover;
+      }
+    </style>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width,initial-scale=1.0" />
+    <link rel="icon" href="<%= BASE_URL %>favicon.ico" />
     <title>
-        <%= htmlWebpackPlugin.options.title %>
+      <%= htmlWebpackPlugin.options.title %>
     </title>
-</head>
+  </head>
 
-<body>
+  <body>
     <noscript>
-    <strong>We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work properly without JavaScript enabled.
-      Please enable it to continue.</strong>
-  </noscript>
+      <strong
+        >We're sorry but <%= htmlWebpackPlugin.options.title %> doesn't work
+        properly without JavaScript enabled. Please enable it to
+        continue.</strong
+      >
+    </noscript>
     <div id="app"></div>
     <!-- built files will be auto injected -->
-</body>
-
-</html>
\ No newline at end of file
+  </body>
+</html>
```

Comment:
I guess you should add a newline at end of each file.

Reply:
Ishita, I did add new lines. When we don't add them, we get a red sign at the end of line which says "No newline at end of file". (This is what I have understood, because I can see newlines in my code files in vs code.) Maybe you can clone it in your machine and check if they are there?